src/tests/specs/workItemTableTest.spec.ts

```diff
@@ -35,12 +31,11 @@ describe('Work Item datatable list', () => {
     expect(await planner.workItemList.getInlineQuickAddClass(c.workItemTitle1)).toContain('disable');
   });
 
-  // The test doesn't work on mock data. Hence, ignore it.
-  // The should work against real database.
-  xit('hideTree and create a work item then work item should be displayed when show tree is selected', async () => {
+  it('hideTree and create a work item then work item should be displayed when show tree is selected', async () => {
     await planner.header.clickShowTree();
     await planner.createWorkItem(c.newWorkItem1);
     expect(await planner.workItemList.hasWorkItem(c.newWorkItem1.title)).toBeTruthy();
+    await planner.quickPreview.notificationToast.untilHidden();
```

Comment:
Do we really need this? We should wait for notification toast to go away only when we try to click the quickpreview close button. In rest of the cases, the notification toast can be ignored.

Reply:
Yes we need this because next step is to click on 'Show Tree' Button and until notification Toast is Hidden .It can't click and thows the exception.